packages/builders/src/node/build/node-build.builder.spec.ts

```diff
@@ -0,0 +1,226 @@
+import { normalize } from '@angular-devkit/core';
+import { TestLogger } from '@angular-devkit/architect/testing';
+import BuildNodeBuilder from './node-build.builder';
+import { BuildNodeBuilderOptions } from './node-build.builder';
+import * as ts from 'typescript';
+import { of } from 'rxjs';
+import { ProgressPlugin } from 'webpack';
+import ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+import CircularDependencyPlugin = require('circular-dependency-plugin');
+
+describe('NodeBuildBuilder', () => {
```

Comment:
I'm not sure if this test is valuable. Its passing doesn't give me much confidence that the code actually works.

Why is having e2e tests not sufficient?

Reply:
e2e tests are sufficient.. they are in the next PR which sets up a project using the schematics.